Resources/views/layouts/default/imports/oro_payment_method_options/layout.html.twig

```diff
@@ -1,6 +1,6 @@
 {% block _payment_methods_paypal_express_widget %}
     <div class="{{ class_prefix }}__form__payment-methods">
-        <img src="{% image '@OroPayPalExpressBundle/Resources/public/default/images/pp-express-checkout-method.png' %}{{ asset_url }}{% endimage %}">
+        <img src="{{ asset('bundles/oropaypalexpress/default/images/pp-express-checkout-method.png') }}">
```

Comment:
@chrisaligent Is it alright be to hard coding this image path to the bundles directory? I feel like it should retain the "magic" path

Reply:
@luke-denton-aligent I just checked with Adam and he's confirmed that it's correct, because it's wrapped in the `asset()` function. None of the core Oro Twig functions use this either:
![image](https://user-images.githubusercontent.com/45612883/52183889-57ffe080-285c-11e9-8c3f-25acd9591286.png)
We've used `asset('bundles/customtheme/subtheme/images/svg/logo.svg')` in the theme too, this code didn't change in 3.1

Reply:
Needs to be done due to the removal of assetic (in favor of webpack) in 3.1.